libr/include/r_core.h

```diff
@@ -648,6 +648,7 @@ R_API int r_core_print_disasm_json(RCore *core, ut64 addr, ut8 *buf, int len, in
 R_API int r_core_print_disasm_instructions (RCore *core, int len, int l);
 R_API int r_core_print_disasm_all (RCore *core, ut64 addr, int l, int len, int mode);
 R_API int r_core_disasm_pdi(RCore *core, int nb_opcodes, int nb_bytes, int fmt);
+R_API bool r_core_disasm_pde(RCore *core, int nb_opcodes, int mode);
```

Comment:
maybe all the other print call sshould return bool? the signature must be consistent with all of them